docs/getting-started/first-trigger.md

```diff
@@ -48,7 +61,7 @@ Now, when we go back to the CloudEvents Player and send an Event, we see that Cl
       kn trigger create cloudevents-player-filter --sink cloudevents-player  --broker example-broker --filter type=some-type
     ```
 
-    If you send a CloudEvent with type "some-type," it will be reflected in the CloudEvents Player UI. Any other types will be ignored by the `Trigger`.
+    If you send a CloudEvent with type "some-type," it will be reflected in the CloudEvents Player UI. Any other types will be ignored by the Trigger.
```

Comment:
If you send a CloudEvent with type "some-type," it is reflected in the CloudEvents Player UI.  The Trigger ignores any other types.